assets/scss/_base.scss

```diff
@@ -918,3 +927,43 @@ section#cncf {
     border-top-left-radius: 0;
   }
 }
+
+//Home page dark-mode
+@media (prefers-color-scheme: dark) {
+  .cid-home main {
+    background-color: $dark-bg-color-2;
+    color:$dark-text-color-1;
+  }
+
+  .cid-home main section:not(#video):not(#cncf) {
+    background-color: $dark-bg-color-1;
+    color:$dark-text-color-1;
+  }
+
+  .cid-home .feature-box div {
+    background-color: $dark-bg-color-2;
+    color:$dark-text-color-1;
+  }
+
+  .cid-home main section:is(#video) #desktopShowVideoButton {
+    background-color: $dark-bg-color-1;
+  }
+
+  .cid-home main section:is(#video) #desktopShowVideoButton:hover {
+    color: $dark-text-color-2;
+  }
+
+  .cid-home main section:is(#video) #desktopKCButton:hover {
+    background-color: $dark-bg-color-1;
+  }
+  
+  .cid-home main section:is(#talkToUs) img {
+    filter: invert(100%);
+    -webkit-filter: invert(100%);
+  }
+  
+  .cid-home main section:is(#cncf) {
+    background-color: $dark-bg-color-2;
+    background-image: url(/images/cncf-white.png);
```

Comment:
Inverting image colors or adjusting background image according to `prefers-color-scheme: dark` is nice addition for the homepage. However, extending this functionality to the entire docs and all its different images would be a challenge. It would be nice if this could be achieved through a [Hugo shortcode](https://gohugo.io/content-management/shortcodes/), dynamically altering image source based on prefers-color-scheme: dark or light. I notice the [issue description(here)](https://github.com/kubernetes/website/issues/37444#issue-1419196244) also hints at employing shortcode for image switching, which aligns well with this suggestion.


For guidance on shortcode, refer to the tips provided at https://gohugo.io/templates/shortcode-templates/